project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_forms.scss

```diff
@@ -93,6 +93,81 @@
   text-transform: uppercase;
 }
 
+// ========================================================================== //
+// Event Form
+// ========================================================================== //
+.node-form {
+  .control-label {
+    clear: both;
+    display: block;
+  }
+
+  .form-control:not(textarea) {
+    color: $color-gray-dark;
+    font-style: italic;
+    background-color: #EBEAE4;
+    border: 0;
+    border-radius: 20px;
+
+    &:focus {
+      box-shadow: 0 0 8px rgba(0, 0, 0, .6);
+    }
+  }
+
+  .document_file {
+    display: none !important;
+  }
+
+  .hidden-input {
```

Comment:
No, it's a [text input](https://github.com/capacity4dev/capacity4more/blob/2370cf7ffc52a45eb09b084ea8ad3e00696290d5/project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/templates/c4m_document_widget.tpl.php#L27). I think it's related to the file naming?
Anyways, it's not showing up in the design - should I remove that element?
![image](https://cloud.githubusercontent.com/assets/13344923/17461864/e09a670e-5ca2-11e6-83a4-6141a63c4251.png)